internal/wrappers/mock/projects-mock.go

```diff
@@ -59,6 +59,20 @@ func (p *ProjectsMockWrapper) Get(params map[string]string) (
 		model = getProjectResponseModel(fmt.Sprintf("%s-id", name), name, filteredTotalCount)
 	case "fake-kics-fail-sast-canceled":
 		model = getProjectResponseModel(fmt.Sprintf("%s-id", name), name, filteredTotalCount)
+	case "existing-project":
+		model = getProjectResponseModel(fmt.Sprintf("%s-id", name), name, filteredTotalCount)
+	case "non-existing-project":
+		model = nil
+	case "error-project":
+		return nil, nil, fmt.Errorf("some error")
+	case "test_project1":
+		model = getProjectResponseModel("1", "test_project1", filteredTotalCount)
+	case "test_project2":
+		model = getProjectResponseModel("2", "test_project2", filteredTotalCount)
+	case "exact_project":
+		model = getProjectResponseModel("3", "exact_project", filteredTotalCount)
```

Comment:
the exact project doesn't initialise the mock with 3 different projects
you can add in getProjectResponseModel
Projects: for in count{ id: i, name: "{name}+{i}"}
then it will give you the test we wanted to test